swagger/generate_swagger.sh

```diff
@@ -15,6 +15,7 @@
 
 ## NOTE: This script is expected to be run from the root 'gate' directory
 
+rm -f swagger/swagger.json
```

Comment:
What about adding this to `.gitignore` instead?

Reply:
This line is to ensure that we generate a new spec each time the script is run. I added a `.gitignore` rule also.